lms/djangoapps/support/static/support/jsx/single_support_form.jsx

```diff
@@ -47,7 +47,17 @@ class RenderForm extends React.Component {
     };
     this.handleClick = this.handleClick.bind(this);
     this.reDirectUser = this.reDirectUser.bind(this);
+    this.handleInputChange = this.handleInputChange.bind(this);
     this.formOnChangeCallback = this.formOnChangeCallback.bind(this);
+    this.handleSearchButton = this.handleSearchButton.bind(this);
+    this.onKeyDown = this.onKeyDown.bind(this);
+    this.handleSuggestionClick = this.handleSuggestionClick.bind(this);
+    this._ignoreBlur = false;
```

Comment:
any particular reason to mark it private?

Reply:
I don't think there's anything good to keep it public.

Reply:
That doesn't answer my question. Unless there is a good reason to make it private or start with underscore, please follow the coding conventions here.

Reply:
Alright! I'll change it.